cmd/verification/main.go

```diff
@@ -170,7 +170,7 @@ func main() {
 			rt := fvm.NewInterpreterRuntime()
 			vm := fvm.NewVirtualMachine(rt)
 			vmCtx := fvm.NewContext(node.Logger, node.FvmOptions...)
-			chunkVerifier := chunks.NewChunkVerifier(vm, vmCtx)
+			chunkVerifier := chunks.NewChunkVerifier(vm, vmCtx, node.Logger.With().Str("component", "chunk_verifier").Logger())
```

Comment:
```suggestion
			chunkVerifier := chunks.NewChunkVerifier(vm, vmCtx, node.Logger)
```
Would be more readable if we just pass the logger here and do the rest internally. Otherwise, we always should make sure that the caller takes care of this part. As seen in the rest of PR, we need to replicate this pattern at least three times on all invocations, which literally makes code less DRY (Don't Repeat Yourself).